cibuildwheel/__main__.py

```diff
@@ -231,6 +243,7 @@ def main() -> None:
         environment=environment,
         dependency_constraints=dependency_constraints,
         manylinux_images=manylinux_images,
+        python_configs=python_configs,
```

Comment:
I'm unconvinced that python_configs should be part of the BuildOptions object, because it's not a user-controllable option. IMO it would be better to call `read_python_configs` from the `get_python_configurations` functions in the platform-specific files.

Reply:
Okay, sounds good.